examples/error-handling/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+// Find all our documentation at https://docs.near.org
+use near_sdk::contract_error;
+use near_sdk::near;
+
+#[contract_error]
+pub enum MyErrorEnum {
+    X,
+}
+
+#[contract_error]
+pub struct MyErrorStruct {
+    x: u32,
+}
+
+#[near(contract_state)]
+#[derive(Default)]
+pub struct Contract {
+    value: u32,
+}
+
+#[near]
+impl Contract {
+    #[handle_result]
+    pub fn inc_handle_result(&mut self, is_error: bool) -> Result<u32, &'static str> {
+        self.value += 1;
+        if is_error {
+            return Err("error in inc_handle_result");
+        } else {
+            return Ok(self.value);
```

Comment:
Let's assume that readers of the examples are familiar with the fact that the last statement in Rust block is the return value, and don't explicitly use `return` statements:

```suggestion
            Err("error in inc_handle_result")
        } else {
            Ok(self.value)
```

P.S. Apply it in all other occurences